.github/workflows/ci.yml

```diff
@@ -27,6 +27,142 @@ jobs:
           name: dojo-bins
           path: bins
 
+  dojo-tests:
+    needs: build
+    runs-on: arc-runner-set
```

Comment:
From where this runner set is coming from?

Reply:
This one is deployed on our visoft cluster.